Sources/Mantis/CropViewController/CropToolbar.swift

```diff
@@ -40,6 +40,20 @@ public class CropToolbar: UIView, CropToolbarProtocol {
         button.setImage(icon, for: .normal)
         return button
     }()
+    
+    private lazy var horizontallyFlipButton: UIButton = {
+        let button = createOptionButton(withTitle: nil, andAction: #selector(horizontallyFlip))
+        let icon = iconProvider?.getClockwiseRotationIcon() ?? ToolBarButtonImageBuilder.horizontallyFlipImage()
```

Comment:
`iconProvider?.getClockwiseRotationIcon()` seems like a copy paste mistake. 

```suggestion
        let icon = iconProvider?.getHorizontallyFlipIcon() ?? ToolBarButtonImageBuilder.horizontallyFlipImage()
```